iteration.go

```diff
@@ -104,22 +146,23 @@ func ConvertIterations(request *goa.RequestData, Iterations []*iteration.Iterati
 }
 
 // ConvertIteration converts between internal and external REST representation
-func ConvertIteration(request *goa.RequestData, iteration *iteration.Iteration, additional ...IterationConvertFunc) *app.Iteration {
-	iterationType := "iterations"
+func ConvertIteration(request *goa.RequestData, itr *iteration.Iteration, additional ...IterationConvertFunc) *app.Iteration {
```

Comment:
yes @kwk fuction parameter name "iteration" was shadowing package "iteration". I had an option to make package name `iterationPkg` and then replace all such instances in the file but I chose to change parameter name instead of imported package name.
And I have made this change only where I used iteration package inside the function not other places.
Is it fine to continue with this name ?